### PR TITLE
Unskip MU adapters tests

### PR DESCRIPTION
**Pull request by dcyriller**
* Unskip adapter tests
* Upgrade ember-cli-blueprint-test-helpers
* Remove the now unused `fs-extra` devDependency

EDIT: it's ready now

### CODE REVIEW

**Comment by runspired** [comment]
@dcyriller thanks for continuing to advance this :)

**Comment by dcyriller** [comment]
@runspired this may be ready now! 🎉

File: package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "common-tags": "^1.8.0",
     "ember-cli": "^3.3.0",
     "ember-cli-app-version": "^3.2.0",
-    "ember-cli-blueprint-test-helpers": "^0.18.3",
+    "ember-cli-blueprint-test-helpers": "github:ember-cli/ember-cli-blueprint-test-helpers",
```
[the remark this answers is not in the document]
Comment (dcyriller) [reply package.json]: I has not been released yet. Just opened an issue in the repo..

[the remark this answers is not in the document]
Comment (dcyriller) [reply package.json]: This should be good now!

**Pull request merged**